doc/source/management/ssl-improvement.rst

```diff
@@ -0,0 +1,42 @@
+.. _percona-server.management.ssl:
+
+================================================================================
+SSL Improvements
+================================================================================
+
+|Persona Server| enables transmitting data in the encrypted form by using the
+TLSv1.2 protocol implemented in the |openssl| toolkit.
+
+|Percona Server| binaries do not link with the |yassl.name| as |MySQL| Community
```

Comment:
Links with *system* OpenSSL
Missing parts from JIRA: "YaSSL security advisories are not relevant for us, system OpenSSL ones are, admin should track those and promptly upgrade as necessary. Functionality differences: link to https://dev.mysql.com/doc/refman/5.6/en/openssl-versus-yassl.html. "